home/templates/home/ukraine.html

```diff
@@ -39,6 +39,15 @@ <h5>STEM, освіта та підвіщення кваліфікації IT с
 
 			<h5>Open Build допомагає малому бізнесу та некомерційним організаціям</h5>
 			Ми надаємо компаніям можливість використовувати онлайн-навчання та програмне забезпечення для відновлення бізнесу за допомогою  маркетингу, продажу та управління бізнесом на місцевому та міжнародному рівні.
+
+			<br>
```

Comment:
You could create a `div` with a specific height to create this space instead of the many line breaks.
Or still have a margin-top/bottom on the neighbouring elements...

Reply:
I'm ready to apply your recommendations, thanks